app/src/main/res/values/strings.xml

```diff
@@ -51,4 +51,6 @@
     <string name="username_min_length_1">"The username should have at least "</string>
     <string name="username_min_length_2">" characters."</string>
     <string name="my_offers">My offers</string>
+    <string name="answer_limit_1">"The answer should be at least "</string>
```

Comment:
I've seen that there exists a way to format strings in `strings.xml`. You can put `"The answer should be at least %d characters."` (but maybe without the quotes) and pass the parameter when you call `getString` (i.e `activity.getString(R.string.answer_limit, Answer.COMMENT_MIN_LENGTH)`)

Reply:
I didn't know that, thank you for showing me. I also updated it in one other place.